.github/workflows/testing.yml

```diff
@@ -101,6 +176,8 @@ jobs:
       with:
         file: ./coverage.xml
   documentation:
+    if: ${{ needs.skips-allowed.outputs.SKIP_DOCUMENTATION == 'false' }}
+    needs: skips-allowed
```

Comment:
What I don't understand is... does this only run if you're enabling `skips-allowed` via `GA Skip Merge Allowed`? Suppose I don't skip anything, does this still run?

Reply:
The `skips-allowed` job always runs regardless of labeling, and will never fail.  I breakout that "merge-able failure" into a separate job since if it fails any dependent jobs would automatically skip.

Since the `outputs` are defined using the job's `env`, the `outputs` are automatically initialized with the `env` values.